src/UserContext.ts

```diff
@@ -30,8 +30,10 @@ export type PortalEnv = "localhost" | "blackforest" | "fairfax" | "mooncake" | "
 const userContext: UserContext = { isTryCosmosDBSubscription: false, portalEnv: "prod" };
 
 function updateUserContext(newContext: UserContext): void {
+  if (newContext.databaseAccount) {
+    newContext.apiType = apiType(userContext.databaseAccount);
```

Comment:
Does this mutate what was passed? Does it matter?

Reply:
userContext => newContext need to replace i have correct in this PR